python/gink/impl/utilities.py

```diff
@@ -33,6 +33,19 @@ def encode_to_hex(string: str) -> str:
     # Adding 0x so we can easily determine if a subprotocol is a hex string
     return "0x" + string.encode("utf-8").hex()
 
+def is_type(obj, type_or_tuple) -> bool:
+    """
+    Returns True if the object is an instance of the type or any type in the tuple of types.
+    This is the default behavior of isinstance in newer versions of Python.
+    """
```

Comment:
isinstance doesn't play well with user defined type generics in older python versions, or at least that's what mypy wants me to think.